integration_tests/tests/match/placementApplications.cy.ts

```diff
@@ -237,6 +242,119 @@ context('Placement Applications', () => {
     })
   })
 
+  it('allows me to review an application', () => {
+    // Given there is a placement request task and placement application in the database
+    const placementApplicationTasks = placementApplicationTaskFactory.buildList(1)
+
+    const document = {
+      'request-a-placement': [{ 'test question 1': 'test answer 1' }, { 'test question 2': 'test answer 2' }],
+    }
+    const placementApplication = placementApplicationFactory.build({ id: placementApplicationTasks[0].id, document })
+    cy.task('stubPlacementApplication', placementApplication)
+
+    cy.task('stubTasks', placementApplicationTasks)
+
+    // When I visit the placementRequests dashboard
+    const listPage = ListPage.visit()
+    listPage.clickPlacementApplications()
+
+    // And I click on the first name
+    listPage.clickPersonName(placementApplicationTasks[0].person.name)
+
+    // Then I should be taken to the review applications page
+    const page = Page.verifyOnPage(ReviewApplicationPage)
+    page.checkPageContents(placementApplication)
+
+    // And when I complete the form
+    page.completeForm()
+    page.clickSubmit()
+
+    // Then I should be taken to the decision page
+
+    const decisionPage = Page.verifyOnPage(ReviewApplicationDecisionPage)
+
+    // And when I complete the form
+
+    cy.task('stubSubmitPlacementApplicationDecision', placementApplication)
+
+    decisionPage.checkRadioByNameAndValue('decision', 'accepted')
+    decisionPage.getTextInputByIdAndEnterDetails('decisionSummary', 'some summary notes')
+    decisionPage.clickSubmit()
```

Comment:
Should we confirm that the endpoint has been called with the data we expect here?